.github/workflows/test.yml

```
@@ -0,0 +1,17 @@
+name: CI
+on:
+  - push
+  - pull_request
+jobs:
+  test:
+    name: Test
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+      fail-fast: false
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Pulsar Editor
+        uses: ./
+      - run: pulsar -v
```

```suggestion
      - run: pulsar -v >> $GITHUB_OUTPUT
      - if: contains( $GITHUB_OUTPUT, 'Pulsar') && contains( $GITHUB_OUTPUT, 'Electron') && contains( $GITHUB_OUTPUT, 'Chrome') && contains( $GITHUB_OUTPUT, 'Node')
        run: |
          echo "::notice $GITHUB_OUTPUT" 
          exit 0
       - if: !contains( $GITHUB_OUTPUT, 'Pulsar') && !contains( $GITHUB_OUTPUT, 'Electron') && !contains( $GITHUB_OUTPUT, 'Chrome') && !contains( $GITHUB_OUTPUT, 'Node')
         run: |
           echo "::error $GITHUB_OUTPUT"
           exit 1
```

Additionally, I noticed that like I mentioned below that even when the Windows command tries and fails to run the command against Pulsar, it doesn't error itself causing the test to pass no matter what.

So with this we can check the output of the command. Seeing if it contains `Pulsar`, `Electron`, `Chrome`, and `Node`. Then check again for not containing these things, and we can go ahead and exit with a bad status to cause the test to fail.

There very likely is a better way to do this, and I haven't been able to test, but just thought I'd add we likely want to check for this exception, and this may work.

---

Windows command doesn't fail, please take a look at my previous comment